firmware/controllers/algo/defaults/default_cranking.cpp

```diff
@@ -23,11 +23,17 @@ void setDefaultCranking() {
 	// After start enrichment
 #if !EFI_UNIT_TEST
 	// don't set this for unit tests, as it makes things more complicated to test
-	engineConfiguration->postCrankingFactor = 1.2;
```

Comment:
setTable(engineConfiguration->postCrankingFactor, 1.2f); for more compatible change?